README.md

```diff
@@ -128,8 +128,50 @@ Docker
 
 - [Developing minetestserver with Docker](doc/developing/docker.md)
 
-We provide a Dockerfile that can be used to build the server.
+We provide Minetest server Docker images using the GitHub container registry.
```

Comment:
This should probably be moved to docs/docker_server.md or something, with a link here